spec/models/sell_opening_flow_spec.rb

```diff
@@ -141,10 +141,10 @@
       end.to change { BitexBot::OpenSell.count }.by(1)
     end
 
-    it 'does not register litecoin buys' do
```

Comment:
This should be something along the lines of "does not register buys from another orderbook"

Reply:
ready